recipes/clara/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+import os
+from conans import ConanFile, tools
+
+
+class ClaraConan(ConanFile):
+    name = "clara"
+    description = "A simple to use, composable, command line parser for C++ 11 and beyond"
+    homepage = "https://github.com/catchorg/Clara"
+    topics = ("conan", "clara", "cli", "cpp11", "command-parser")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSL-1.0"
+    no_copy_source = True
+
+    _source_subfolder = "source_subfolder"
+
+    def configure(self):
+        self.output.warn("clara is unmaintained. Consider updating to lyra/1.2")
```

Comment:
Clara and Lyra are interchangeable?

Reply:
Not interchangeable, but it is maintained.
There is a banner on clara's github page pointing to lyra.

Reply:
And we have Lyra in CCI, so it will not affect the users who are looking for Lyra after Clara.

Reply:
So if the project is nearly dead, why would someone use it? Are libraries relying on this one? Why did you *prioritize* creating this recipe and not others?

Reply:
fair enough, only for legacy propose I think.